README.md

```diff
@@ -44,6 +44,7 @@ Kraken supports pod, node, time/date and [litmus](https://github.com/litmuschaos
 
 - [Cluster Shut Down Scenarios](docs/cluster_shut_down_scenarios.md)
 
+- [Namespace Scenarios](docs/namespace_scenarios.md)
```

Comment:
@paigerube14 we might have missed the docs around the namespace delete scenarios.

Reply:
Added it in!